### DISCUSSION TITLE
pada geom_line, garisnya tidak keluar

### DISCUSSION THREAD

**Discussion by annerina84** [teamalgoritma/community#552] [Q&A] [answered]
Halo, mhn bantuannya, kenapa garis pada geom_line tidak keluar di plot
```{r}
plot3 <- ggplot(gempa_tahunan, aes(x=tahun, y= Frekuensi_Gempa))+
  geom_line(col="blue") +
  geom_point(aes(text=Frekuensi_Gempa), col="black") +
  #scale_y_continuous() +
 # scale_x_continuous(breaks = seq(0,23,1)) +
  labs(
    title = "Aktivitas Gempa di Di Indonesia Periode Tahun 2009-2022",
    x = "Tahun",
    y = "Kejadian Gempa"
  ) +
  theme_minimal()

ggplotly(plot3, tooltip = "text")
```
apakah ada yang kurang pada code di atas.

<img width="645" alt="image" src="https://user-images.githubusercontent.com/120439978/220011840-f5fc3cde-9ad1-4ba2-b4e3-acbd0bf3f278.png">

**Comment by finesaaa** [comment, marked as answer]
Dear Bu @annerina84,
Sebaiknya Ibu memastikan kedua kolom yang dipakai x dan y adalah numeric, jika ada yang factor silahkan diubah menjadi numeric sebagai berikut
```
gempa _tahunan$tahun <- as.numeric(as.character(gempa $tahun))
```
